benchmarks/nightly.sh

```diff
@@ -141,11 +161,14 @@ for testname in inference training; do
       for tier in '' 1; do
 	# Note: these (as well as $tier) can be null; read them with ${FOO?}
 	# instead of ${FOO:?}.
-	TITLE_PFX=
+        TITLE_PFX=
```

Comment:
what does PFX stand for?

Reply:
PREFIX. I've updated the PR with s/PFX/PREFIX/.